src/app/boot/app_controller.nim

```diff
@@ -98,9 +100,9 @@ proc mainDidLoad*(self: AppController)
 #################################################
 
 proc connect(self: AppController) =
-  self.statusFoundation.status.events.once("nodeStopped") do(a: Args):
-    # TODO: remove this once accounts are not tracked in the AccountsModel
-    self.statusFoundation.status.reset()
```

Comment:
What did `reset` do before?

Reply:
In the old code it just was deleting the list of generated accounts, but that is not applicable now since we have totally different logic in refactored code, and there is nothing to be deleted in regards of that.